CONTRIBUTING.md

```diff
@@ -47,30 +49,37 @@ This allows other members of the Nipype development team to confirm that you are
 
 **2. [Fork][link_fork] the [Nipype repository][link_nipype] to your profile.**
 
-This is now your own unique copy of Nipype.
+This is now your own unique copy of Nipype repository.
 Changes here won't affect anyone else's work, so it's a safe space to explore edits to the code!
 
-Make sure to [keep your fork up to date][link_updateupstreamwiki] with the master repository.
+You can clone your Nipype repository in order to create a local copy of the code on your machine.
+In your local Nipype directory, run `pip install -e .[dev]`.
+This will add your version of nipype to your local python environment and
+install dependencies needed for development.
```

Comment:
I would add the justification first.

```Markdown
To install your version of Nipype, and the dependencies needed for development,
in your Python environment, run `pip install -e ".[dev]"` from your local Nipype
directory.
```

(I'm adding the quotes because zsh tries to interpret the brackets. I'm not sure if any other shells do, but the above line is slightly safer.)